src/components/projects-summary/projects-summary.html

```diff
@@ -55,10 +49,13 @@ <h3 class="project-tile-header">
           <a href="" ng-click="$ctrl.handleProjectClick(project)" class="project-title">{{project | displayName}}</a>
         </h3>
         <p class="project-date">
+          <span ng-if="project | displayName : true"><span ng-bind-html="project.metadata.name | highlightKeywords : keywords"></span> &ndash;</span>
```

Comment:
Remove use of `highlightKeywords` since we have no filtering on this page (here are below).